server/settings/general.py

```diff
@@ -36,6 +70,11 @@ class GeneralSettingsModel(BaseSettingsModel):
 
 DEFAULT_GENERAL_SETTINGS = {
     "add_self_publish_button": False,
+    "set_rop_output": {
+        "enabled": True,
+        "expand_vars": False,
+        "default_output_dir": "$HIP/ayon/`chs('AYON_productName')`"
```

Comment:
Let's use `"` in the houdini expression by default. I rarely see anyone use `'` there.
```suggestion
        "default_output_dir": "$HIP/ayon/`chs(\"AYON_productName\")`"
```

Reply:
https://github.com/ynput/ayon-houdini/pull/200/commits/47f09fdd093c25372baf39ec6855244060a566d1